src/components/calculator.js

```diff
@@ -1,39 +1,61 @@
 import React from 'react';
+import calculate from '../logic/calculate';
+// import calculate from '../logic/calculate';
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Kindly remove commented code. It makes your codebase messy and hard to read.

- [ ] Kindly remove unnecessary linter disabling here.